scipy/signal/_waveforms.py

```diff
@@ -288,6 +289,11 @@ def chirp(t, f0, t1, f1, method='linear', phi=0, vertex_zero=True):
         This parameter is only used when `method` is 'quadratic'.
         It determines whether the vertex of the parabola that is the graph
         of the frequency is at t=0 or t=t1.
+    complex : bool, optional
+        Only available if `method` is ``'linear'`` or ``'quadratic'``.
```

Comment:
Should we raise an error if this is not the case?

Reply:
I'll be updating the docstring and comments. All should be fixed by today